src/app/platform/keys/create/page.tsx

```diff
@@ -24,6 +24,12 @@ const Page = () => {
       return;
     }
 
+    if (name.length < 3) {
```

Comment:
We are duplicating validation here. We already have `zod` based schema validation at `src/lib/schema/apikey.ts`. Say in future, we change the validation criteria to enforce at least 4 chars for name. We will update `src/lib/schema/apikey.ts` because that where we define the schema. If we forget changing the logic here then we have two different validation logic.

As a best practice, we should have only 1 place for validation logic which is `src/lib/schema/apikey.ts`. See if we can use the same `zod` schema for validation. Else remove the validation logic from here